pontoon/insights/static/js/insights_tab.js

```diff
@@ -0,0 +1,953 @@
+            },
+            legendCallback: Pontoon.insights.customLegend(chart),
+            tooltips: {
+              mode: 'index',
+              intersect: false,
+              borderColor: '#7BC876',
+              borderWidth: 1,
+              caretPadding: 5,
+              xPadding: 10,
+              yPadding: 10,
+              itemSort: function (a, b) {
+                // Dataset order affects stacking, tooltip and
+                // legend, but it doesn't work intuitively, so
+                // we need to manually sort tooltip items.
+                if (a.datasetIndex === 2 && b.datasetIndex === 1) {
+                  return 1;
+                }
+              },
+              callbacks: {
+                label: function (items, chart) {
+                  const human = chart.datasets[1].data[items.index];
+                  const machinery = chart.datasets[2].data[items.index];
+
+                  const label = chart.datasets[items.datasetIndex].label;
+                  const value = items.yLabel;
+                  const base = label + ': ' + nf.format(value);
+
+                  switch (label) {
+                    case 'Completion':
+                      return base + '%';
+                    case 'Human translations':
+                    case 'Machinery translations': {
+                      const pct = Pontoon.insights.getPercent(
+                        value,
+                        human + machinery,
+                      );
+                      return `${base} (${pct} of all translations)`;
+                    }
+                    default:
+                      return base;
+                  }
+                },
+              },
+            },
+            scales: {
+              xAxes: [
+                {
+                  stacked: true,
+                  type: 'time',
+                  time: {
+                    displayFormats: {
+                      month: 'MMM',
+                    },
+                    tooltipFormat: 'MMMM YYYY',
+                  },
+                  gridLines: {
+                    display: false,
+                  },
+                  offset: true,
+                  ticks: {
+                    source: 'data',
+                  },
+                },
+              ],
+              yAxes: [
+                {
+                  id: 'completion-y-axis',
+                  position: 'right',
+                  scaleLabel: {
+                    display: true,
+                    labelString: 'COMPLETION',
+                    fontColor: '#FFF',
+                    fontStyle: 100,
+                  },
+                  gridLines: {
+                    display: false,
+                  },
+                  ticks: {
+                    beginAtZero: true,
+                    max: 100,
+                    stepSize: 20,
+                    callback: function (value) {
+                      return value + ' %';
+                    },
```

Comment:
Using `Intl.NumberFormat` would make sure that we don't end up with differences in how we format percentage values like this: There's a space here, while there isn't in the previous.